### PR TITLE
Reaction emoji pool

### LINKED ISSUE #192: Bot reaction to correct 'rade should select from a pool of potential emojis
Currently it's just 👏 but we should make it a pool of emojis similar to the 'Congrats' message!

### PR DESCRIPTION
**Pull request by michael-robbins**
This fixes #192

### CODE REVIEW

File: emojirades/analytics/time_range.py [hunk 1]
```diff
@@ -4,15 +4,16 @@
 
 
 class TimeRange:
-    pendulum_unit = {
-        TimeUnit.WEEKLY: "week",
-        TimeUnit.MONTHLY: "month"
-    }
+    pendulum_unit = {TimeUnit.WEEKLY: "week", TimeUnit.MONTHLY: "month"}
```
Comment (phelios) [inline emojirades/analytics/time_range.py:7]: I prefer the previous version. But I guess we have to follow Black.

File: emojirades/commands/gamestate_commands/inferred_correct_guess_command.py [hunk 2]
```diff
@@ -71,7 +82,7 @@ def execute(self):
         else:
             emoji = random.choice(self.other_emojis)
 
-        emoji = f" {emoji}"
+        emoji = f" :{emoji}:"
```
Comment (phelios) [inline emojirades/commands/gamestate_commands/inferred_correct_guess_command.py:85]: so erm.. should these be reflected in the other class `correct_guess_command.py` as well?

Comment (michael-robbins) [reply emojirades/commands/gamestate_commands/inferred_correct_guess_command.py:85]: That's a dam good question

Comment (michael-robbins) [reply emojirades/commands/gamestate_commands/inferred_correct_guess_command.py:85]: ok pushed!

File: emojirades/analytics/time_range.py (hunk 1 above)
Comment (michael-robbins) [reply emojirades/analytics/time_range.py:7]: Yeah black...

**Pull request merged**